Security/day01/first_part/4-Log/log.txt

```diff
@@ -0,0 +1,24 @@
+Dec 8 15:11:19 server kernel: [856938.128348] TCP: Possible SYN flooding on port 80. 
```

Comment:
useless file ?

Reply:
No, it will be used for resolving the challenge